client/rest-high-level/src/test/java/org/elasticsearch/client/ml/inference/TrainedModelConfigTests.java

```diff
@@ -52,15 +52,22 @@ protected boolean supportsUnknownFields() {
 
     @Override
     protected TrainedModelConfig createTestInstance() {
+        List<String> tags = null;
+        if (randomBoolean()) {
+            int tagsCount = randomIntBetween(0, 5);
+            tags = new ArrayList<>(tagsCount);
+            for (int i = 0; i < tagsCount; i++) {
+                tags.add(randomAlphaOfLength(10));
+            }
+        }
         return new TrainedModelConfig(
             randomAlphaOfLength(10),
             randomAlphaOfLength(10),
             Version.CURRENT,
             randomBoolean() ? null : randomAlphaOfLength(100),
             Instant.ofEpochMilli(randomNonNegativeLong()),
-            randomBoolean() ? null : randomNonNegativeLong(),
-            randomAlphaOfLength(10),
             randomBoolean() ? null : TrainedModelDefinitionTests.createRandomBuilder().build(),
+            tags,
```

Comment:
nit, `Stream.generate` is awesome.
```suggestion
            randomBoolean() ? null : Stream.generate(() -> randomAlphaOfLength(10)).limit(randomIntBetween(0, 5).collect(Collectors.toList()),
```

Reply:
Nice one indeed! Done!